plans/all.fmf

```diff
@@ -13,6 +13,7 @@ prepare:
     - python3-boto3
     - python3-flake8
     - python3-paramiko
+    - python3-boto3
```

Comment:
(nipick^3) this change shold probably be part of commit abde26512f9d3238214f913458a2e3bd607f4283

Reply:
(it's actually a duplicate)

Reply:
Done.